hyperspy/drawing/mpl_he.py

```diff
@@ -106,6 +107,7 @@ def plot_navigator(self):
                         axis.connect(imf.update)
 
             imf.title = self.signal_title + ' Navigator'
+            imf.scalebar_color = self.scalebar_color
```

Comment:
This line is also necessary, or your navigation images will not have the scale color that you specified.